pkg/imagestore/imagestore.go

```diff
@@ -235,6 +244,10 @@ func isoFileName(imageType, openshiftVersion, version, arch string) string {
 	return fmt.Sprintf("rhcos-%s-%s-%s-%s.iso", imageType, openshiftVersion, version, arch)
 }
 
+func buildRootfsURL(baseURL string, arch, version string) string {
+	return fmt.Sprintf("%s/boot-artifacts/rootfs?arch=%s&version=%s", baseURL, arch, version)
```

Comment:
Why remove the URL builder? That will handle things like if someone puts a slash at the end of the url or if the version contains url unsafe characters for some reason.

Reply:
yeah good point, added.